docs/cloudformation.json

```diff
@@ -870,7 +870,10 @@
     },
 
     "CustomResourcesQueue": {
-      "Type": "AWS::SQS::Queue"
+      "Type": "AWS::SQS::Queue",
+      "Properties": {
+        "VisibilityTimeout": "1800"
```

Comment:
Yeah good point. Probably the best thing to do would be to leave the default, and then use the [ChangeMessageVisibility](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ChangeMessageVisibility.html) api call to increase it when needed.

I'll revert the current change for now, since the de-duping is sufficient to fix the current issue.